.github/workflows/checks.yml

```
@@ -37,23 +37,28 @@ jobs:
           echo DJANGO_DEBUG="False" >> .env
           echo DJANGO_SECRET_KEY="${{ secrets.DJANGO_SECRET_KEY }}" >> .env
 
-      - name: Fix permission for logs directory
+      - name: Fix permission
         run: |
-          mkdir -p ./backend/codesirius/logs
-          chmod 777 ./backend/codesirius/logs
+          chmod -R 777 backend/codesirius
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)